CONTRIBUTING.md

```diff
@@ -1,20 +1,69 @@
-# Contribution Guide
+# Contributing to AdoptOpenJDK/infrastructure
 
-The infrastructure project contains:
+Thank you for your interest in AdoptOpenJDK/infrastructure!
+
+We welcome and encourage all kinds of contributions to the project, not only
+code. This includes bug reports, user experience feedback, assistance in
+reproducing issues and more.
+
+The AdoptOpenJDK/infrastructure project contains:
 
 1. The [Ansible Playbooks](ansible/playbooks) for bootstrapping the build and test hosts (including a way to test Ansible)
 1. The overriding [Documentation](docs) for the build farm
 
-To make a change please:
+
+## Submitting a contribution to AdoptOpenJDK/infrastructure9
+
+You can propose contributions by sending pull requests (PRs) through GitHub.
+Following these guidelines will help us merge your pull requests smoothly:
+
+1. Your pull request is an opportunity to explain both what changes you'd like
+   pulled in, but also _why_ you'd like them added. Providing clarity on why
+   you want changes makes it easier to accept, and provides valuable context to
+   review.
+
+2. Follow the commit guidelines found below.
+
+3. We encourage you to open a pull request early, and mark it as "Work In
```

Comment:
Yeah nice to formalise this - also makes it more obvious that someone is already working on something instead of risking duplicate effort :-)